extensions/2.0/Vendor/EXT_lights_image_based/schema/glTF.EXT_lights_image_based.schema.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "EXT_lights_image_based glTF extension",
+    "type": "object",
+    "properties": {
+        "lights": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "$ref": "light.schema.json"
+            },
+            "default": []
```

Comment:
To align with the core spec and `KHR_lights_punctual`, `"default": []` should be changed to `"minItems": 1`.
Also, `lights` should be a required field to disallow empty extension object:
```json
{
  "asset": {
    "version": "2.0",
    "extesnionsUsed": ["EXT_lights_image_based"]
  },
  "extensions": {
    "EXT_lights_image_based": {}
  }
}